docs/usage/security-modes.md

```diff
@@ -0,0 +1,86 @@
+# Security Modes
+
+## Overview
+
+This section describes the different options in terms of **connectivity security modes** available in Liqo, which can be used to impose some restrictions to inter-cluster and intra-cluster traffic.
+
+To better explain the concept, pictures and tables will use the following terminology:
+
+* **Cx**: physical Cluster (e.g. Cluster 1).
+* **Px**: Pod (e.g., Pod 1).
+* **Sx**: Service (e.g., a Cluster IP service), pointing to one or more pods. Given the nature of Liqo, the service endpoints associated to this service can be pods in either clusters.
+* **Ix**: Internet connectivity. For instance, I1 is the Internet connectivity provided by Cluster 1.
+* **Virtual cluster**: refers to the combination of a physical cluster along with all its extensions in other clusters, which are created with Liqo. A virtual cluster includes all the physical resources of the local cluster, and all the resources it borrows from the remote cluster.
+* **Offloaded namespace**: namespace that is expanded across the virtual cluster, enabling the seamless utilization of resources beyond the physical cluster boundaries. An offloaded namespace does not include all the resources of the local cluster, but only the resources belonging to the namespace that has been offloaded (in both local and remote cluster).
+
+Furthermore, yellow cells in the connectivity matrices highlight the differences compared to the default behaviour (i.e., _full pod-to-pod_ connectivity).
+
+## Security modes
+
+Liqo provides two security modes.
+
+### Full pod-to-pod connectivity (default mode)
+
+In this mode, no restrictions are enforced in terms of connectivity, and all pods of the local cluster can connect to all pods of the remote cluster (and vice versa), for all namespaces including non-offloaded ones.
+This mode can be useful when the clusters trust each other (and/or belong to the same administrative domain) and there is no need to restrict the connectivity.
```

Comment:
```suggestion
This mode can be useful when the clusters trust each other (and/or belong to the same administrative domain) and there is no need to restrict the connectivity.
This may happen when both clusters are under control of the same organization, and Liqo is used simply to "cross the boundaries" among distinct clusters (e.g., because clusters are running in different geographical regions), providing a single point of control for many geographically distributed clusters.
```